package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hubble/request",
-  "version": "2.0.1",
+  "version": "2.1.0",
```

Comment:
I've tried publishing my changes to npm using the instructions in the README but they aren't showing up when I upgrade in `site`. It looks like I've bumped the version but the new version doesn't contain my changes. Was I not supposed to change the version manually in the package json? I'm unsure why this hasn't worked.

Reply:
Never mind. This did actually work, I was just using dev containers wrong.